ConnectionManager.lib.nut

```diff
@@ -1,15 +1,36 @@
+// The MIT License (MIT)
```

Comment:
All the files: [LICENSE](./LICENSE), [Example](https://github.com/electricimp/ConnectionManager/blob/master/example/ConnectionManager.example.nut)  and the the library file need to be updated.